tests/test_auth.py

```diff
@@ -0,0 +1,83 @@
+from django.test import TestCase
+from rest_framework.test import APIClient
+from rest_framework import status
+
+from authors.apps.authentication.models import User
+
+class ViewTestCase(TestCase):
+    """Test suite for the api views."""
+
+    def setUp(self):
+        """Define the test client and other test variables."""
+
+        self.user_data = {
+            "username":"nerd",
+            "email":"nerd@nerd.com",
+            "password":"secret"
+            }
+
+        # Initialize client
+        self.client = APIClient()
+
+    def test_api_can_create_a_user(self):
+        """Test the api can succefully create a user."""
```

Comment:
This line reads "succefully" instead of "successfully". Please update.

Reply:
Thanks for pointing that out. I will correct it right away.